### DISCUSSION TITLE
D/K/G Overlap

### DISCUSSION THREAD

**Discussion by MusicIsTheBest** [MKT-Playbook/Hub#14] [Feature requests]
When you look at a CARD's best tracks, that number is the same even when you prefer another CARD. I don't use spreadsheets, but I would like to see overlap and I think I probably have to find one that does show overlap but I'll have to learn how to export the data. I love the app and overlap is the only blind spot I notice.

**Comment by TitusPosiadly** [comment]
Hey there, yeah thanks for the idea. I will implement a better, in-depth coverage tool hopefully sometime soon. I will just have to find the time to work on it again. Still quite busy with my main job atm. But I noted it down and pushed the coverage tool up in priority. Cheers!

**Comment by TitusPosiadly** [comment]
Oh and regarding the export function, you can find the export tool in the settings. The format for most of the popular spreadsheets is already stored in the app and can be accessed through the templated. Depending on our location you will likely have to use a specific delimiter that separated the different values. Just try out the different delimiters and one of them should work. If you found another spreadsheet that isn't yet stored as a template let me know and I will add it.

**Comment by MusicIsTheBest** [comment]
I'm trying to export my MKTPLAYBOOK data for the Bam spreadsheet but I
don't understand the tech sequence. I just get a crytic table of values.
The problem is you have to use multiple apps and no one explains how to do
it. I know this is a power feature once I get over the learning curve and
would address the overlap. I've tried to avoid spreadsheets, but it'll be
worth it to learn.

**Reply by TitusPosiadly** [reply]
The Bam spreadsheet has a page for importing data:
![SCR-20221120-t4p](https://user-images.githubusercontent.com/79757836/202923446-a3751e57-a1f3-4032-8597-9751db4826fa.png)


The easiest way is to copy the export data to your clipboard and paste the values on this page:
![SCR-20221120-t5f](https://user-images.githubusercontent.com/79757836/202923495-0482c224-3df9-4bcf-8106-b574bbae93bf.png)

If you are on a PC you can save the data as .csv or .txt, send that file to your pc, and copy/paste the values from there.

**Comment by MusicIsTheBest** [comment]
Thank you for your email. This walkthrough or one similar needs to be on
the MKT subreddit. Once I understood that I was copying/pasting text from
the CSV into ANOTHER speadsheet, it clicked. Every MKT video I watched was
just about how great they are, not actually how to import/export. Took me
probably two solid hours digging around. I'm not the only one.
